config/develop/glue-job-S3ToJsonS3.yaml

```diff
@@ -0,0 +1,16 @@
+template_path: glue-spark-job.j2
+stack_name: glue-job-S3ToJsonS3
+#{% set BranchOrTagName='main' %}
```

Comment:
Does this job have dependencies?

Reply:
It does -- these are not called explicitly in this config but implicitly